hapi-loader/pom.xml

```diff
@@ -189,12 +189,6 @@
             <version>1.8.4</version>
         </dependency>
```

Comment:
Is the dependency above still necessary?

Reply:
nope, good catch :+1: